SoObjects/Appointments/SOGoCalendarComponent.m

```diff
@@ -549,37 +549,42 @@ - (BOOL) expandGroupsInEvent: (iCalEvent *) theEvent
           pool = [[NSAutoreleasePool alloc] init];
         }
 
-      group = [SOGoGroup groupWithEmail: [currentAttendee rfc822Email]
-                               inDomain: domain];
-      if (group)
-	{
+      dict = [[SOGoUserManager sharedUserManager] contactInfosForUserWithUIDorEmail: [currentAttendee rfc822Email]];
```

Comment:
Respect the domain by using `[SOGoUserManager contactInfosForUserWithUIDorEmail:inDomain:]`.

Reply:
Resolved in commit 948c89f